### ISSUE TITLE
No Splash on New Image

### ISSUE THREAD

**Issue by girijesh18** [nitsourish/car-damage-detection-using-CNN#1]
Hi Sir,
I followed your post and able to train on my own custom dataset of damaged cars. My trained weights work fine on annotated data. Trained weights are able to produce splash on the annotated data but when I am trying to detect damage on un-annotated damage car it is not able to detect any damage and produce splash as well. I have trained for 128 epoch with 164 steps on 300 annotated images. Can you please tell what could be a possible reason for it. Any help would be highly appreciated.

**Comment by AnithaRajalakshmi** [comment]
> Hi Sir,
> I followed your post and able to train on my own custom dataset of damaged cars. My trained weights work fine on annotated data. Trained weights are able to produce splash on the annotated data but when I am trying to detect damage on un-annotated damage car it is not able to detect any damage and produce splash as well. I have trained for 128 epoch with 164 steps on 300 annotated images. Can you please tell what could be a possible reason for it. Any help would be highly appreciated.

Whether you gave un-annotated new image for prediction or for training and Where have you included the un-annotated new images??

**Comment by girijesh97** [comment]
Yes, I provided the un-annotated image for prediction but the model was unable to produce any bounding box and splash. Later, I worked on a similar problem and the same approach yield me results. I haven't included image anywhere I just passed the image path as from the command line as an argument.
